### PR TITLE
Add SelectedIds props for controlled selection

### PR DESCRIPTION
**Pull request by kpustakhod**
Added new property SelecttedIds for controlled selection of nodes at any point of time.

### CODE REVIEW

**Comment by kpustakhod** [comment]
@lissitz can you please take a look? Can't add anyone to reviewers list.

File: src/TreeView/index.tsx [hunk 1]
```diff
@@ -367,6 +369,7 @@ const treeReducer = (
 
 interface IUseTreeProps {
   data: INode[];
+  controlledIds?: number[];
```
Comment (mellis481) [inline src/TreeView/index.tsx:372]: @kpustakhod You had named this prop `checkedIds` in our repo...

Comment (kpustakhod) [reply src/TreeView/index.tsx:372]: @mellis481 I named it `checkedIds` first, then it was renamed to `controlledIds`.

Comment (mellis481) [reply src/TreeView/index.tsx:372]: @kpustakhod I would expect our repo and this repo to be identical. Why was it not updated in our repo?

Comment (kpustakhod) [reply src/TreeView/index.tsx:372]: Prop names were updated in separate PR - https://github.com/vertexinc/react-accessible-treeview/pull/16/files

**Pull request closed without merging**